wheels/image/Makefile

```diff
@@ -0,0 +1,287 @@
+BASE_TAGS	= debian6 debian7 debian8 debian9 ubuntu12.04 ubuntu14.04 centos6 centos7
+BASE_TAGS_32	= $(BASE_TAGS:=-32)
+WHEEL_TAGS 	= $(BASE_TAGS:=-wheel)
+WHEEL_TAGS_32 	= $(WHEEL_TAGS:=-32)
+ALL_TAGS	= $(BASE_TAGS) $(BASE_TAGS_32) $(WHEEL_TAGS) $(WHEEL_TAGS_32)
+CLEAN_TAGS	= $(ALL_TAGS:=.clean)
+
+DEBIAN_MIRROR	?= http://carroll.aset.psu.edu/pub/linux/distributions/debian/
+UBUNTU_MIRROR	?= http://carroll.aset.psu.edu/pub/linux/distributions/ubuntu/
+
+DEBIAN_PKGS	= build-essential ca-certificates curl zlib1g-dev libssl-dev libbz2-dev python-yaml
+DEBIAN_RUN	= apt-key adv --keyserver pgp.mit.edu --recv-keys 8B48AD6246925553 \\\&\\\& apt-get -qq update \\\&\\\& apt-get install --no-install-recommends -y $(DEBIAN_PKGS)
+DEBIAN_RUN_32	= apt-key adv --keyserver pgp.mit.edu --recv-keys 8B48AD6246925553 \\\&\\\& linux32 apt-get -qq update \\\&\\\& apt-get install --no-install-recommends -y $(DEBIAN_PKGS)
+CENTOS_PKGS	= curl zlib-devel openssl-devel bzip2-devel PyYAML
+CENTOS_RUN	= yum groupinstall -y "Development Tools" \\\&\\\& yum install -y $(CENTOS_PKGS)
+CENTOS_RUN_32	= linux32 yum groupinstall -y "Development Tools" \\\&\\\& linux32 yum install -y $(CENTOS_PKGS)
+WHEEL_VERSION	= 35e48cf7c4af
+
+all: $(BASE_TAGS) $(BASE_TAGS_32) $(WHEEL_TAGS) $(WHEEL_TAGS_32)
+
+clean: $(CLEAN_TAGS)
+
+distclean:
+	sudo rm -rf $(ALL_TAGS)
+
+$(ALL_TAGS): %: %/Makefile
+	$(MAKE) -C $@ image-id
+
+$(CLEAN_TAGS):
+	$(MAKE) -C $(basename $@) clean
+
+debian6/Makefile: image-Makefile
+	mkdir -p $(@D)
+	sed -e 's#REPLACE_TAG#natefoo/$(@D)#' \
```

Comment:
Funny, this is a replacement of what I'd originally wrote, which used Makefile variables...

I was originally trying to have make generate the Makefiles, but that proved to be pretty difficult to get make to cooperate with. But I am no make expert.